### ISSUE TITLE
[TW-178] The 'has' attribute modifier should be polymorphic, understanding dependency id/uuid

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#227]
_Paul Beckingham on 2013-08-21T10:34:56Z says:_ 

 Adding polymorphism to the 'has' attribute modifier lets us query like this:

<pre>
task depends.has:22 list
</pre>

This will override the substring search with something more like the Python:

<pre>
if 3 in [1, 2, 3, 4]:
</pre>

Note that this should work for both id and UUID values.

Thanks to SynrG and King_DuckZ.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-08-21T10:34:56Z
Modified: 2015-02-09T14:33:49Z
```

**Comment by taskwarrior** [comment]
_Renato Alves on 2014-09-22T14:12:50Z says:_ 

 Tests for "dep.contains" present in TW-1262.

**Comment by smemsh** [comment]
@tbabej I believe the "fixes" comment from the above commit is erroneous, it doesn't seem to relate to this issue.  Indeed, the suggestion from this issue would sure be a nice thing to have, I think it would fix #2193 ?

**Comment by tbabej** [comment]
Yeah, that's an accidental closure because the commit referred to a different tracking system. Good catch!
